src/f32/funcs.rs

```diff
@@ -139,6 +168,69 @@ pub(crate) mod sse2 {
         _mm_or_ps(result, _mm_castsi128_ps(test))
     }
 
+    #[inline]
+    #[allow(non_snake_case)]
+    pub(crate) const fn _MM_SHUFFLE(z: u32, y: u32, x: u32, w: u32) -> i32 {
+        // core::arch::x86_64::_MM_SHUFFLE requires nightly
+        ((z << 6) | (y << 4) | (x << 2) | w) as i32
+    }
```

Comment:
I prefer the binary literals. I used to use `_MM_SHUFFLE` but actually found it harder to follow what was going on so migrated to binary literals instead.

Reply:
I've changed this to use binary literals